### DISCUSSION TITLE
Connection Status Indicator in Recent Sessions

### DISCUSSION THREAD

**Discussion by pixelstuff** [rustdesk/rustdesk#4730] [Feature Request]
In the Recent Sessions and Favorites tabs can there be some type of online indictor? It would be helpful for situations like a reboot when installing updates to know when the system it is back up, instead of needing to repeatedly attempt a connection until it finally goes through.

**Comment by ghost** [comment]
There already is - the green or orange dot is the online / offline indicator.

**Reply by pixelstuff** [reply]
Is this an upcoming v1.2.0 feature?

**Reply by ghost** [reply]
I've never used 1.19 so it must be a 1.2 feature.

**Comment by egpgit** [comment]
All my peers have orange indicator and I know that the peers are online, so it doesn't  work for me, I am using 1.2.0 Build date 19/06/2023 00:38

**Comment by SergeyMy** [comment]
I confirm
green or orange dot - no longer reflect the real state.
1.2.0 Build date  2023-06-22 18:38

ADD **rustdesk-server changed** 
1.1.8 the indicator seems to work :)
https://github.com/rustdesk/rustdesk-server/releases/tag/1.1.8
